frontend/src/components/editProject/EditProjectOverview.tsx

```
@@ -77,20 +77,20 @@ const useStyles = makeStyles<Theme, { image?: string }>((theme) => ({
-  openCategoriesDialogButton: {
-    marginTop: theme.spacing(1),
+  sectorField: {
+    marginTop: theme.spacing(3),
   },
 }));
 
 type Args = {
   project: Project;
-  handleSetProject: Function;
+  handleSetProject: (project: Project) => void;
   smallScreen: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)